components/docs/introduction-deploy.js

```diff
@@ -19,6 +21,31 @@ const example = {
   }
 }
 
+const AmpFallback = ({ children }) => {
+  const isAmp = useAmp()
+  if (isAmp)
+    return (
+      <>
+        <Head>
+          <script
+            async
```

Comment:
```suggestion
            key="amp-iframe"
            async
```

It's important to always key AMP script tags.

Reply:
Updated